docs/client_reference.rst

```diff
@@ -59,6 +54,15 @@ Usage example::
    :param aiohttp.helpers.BasicAuth auth: BasicAuth named tuple that represents
                                           HTTP Basic Auth (optional)
 
+   :param request_class: Request class implementation. ``ClientRequest``
+                         by default.
+
+   :param response_class: Response class implementation.
+                          ``ClientResponse`` by default.
+
+   :param ws_response_class: WebSocketResponse class implementation.
+                             ``ClientWebSocketResponse`` by default.
+
```

Comment:
Add `.. versionadded:: 0.16` with mention of `ws_response_class` parameter.
Also you may add `..versionchanged:: 0.16` to reflect changing default values for `request_class` and `response_class`.